### DISCUSSION TITLE
Should there be a variant between `text-4xl` and `text-5xl`

### DISCUSSION THREAD

**Discussion by cysabi** [tailwindlabs/tailwindcss#10759] [Ideas]
I feel like the jump between those 2 values is a bit drastic.
Obviously it would be impractical to call it `text-4.5xl`, and it's equally impractical to shift everything down the scale and start a new major version.

I find myself adding a `text-4.5xl` (`2.5rem`) into my config fairly consistently.

Here's what I mean:
- the pixel jump between `text-xs` to `text-lg` is `2px` (4 values)
- the pixel jump between `text-lg` to `text-3xl` is `4px` (3 values)
- the pixel jump between `text-3xl` to `text-4xl` is `6px` (1 value)
- **the pixel jump between `text-4xl` to `text-5xl` is `12px`, that's +6 instead of +2**

To me that feels a bit sudden, but I might be the only one 🤷 
Let me know what you think

**Comment by james-robson** [comment]
Over a year later, but bumping to add that this is a problem for me. I am porting an existing UI to Tailwind, so I have to find the closest possible values to our existing UI and ideally avoid bloating our config by extending with lots of custom types. I was similarly surprised to find the large gap between 4xl and 5xl.

My initial thought was exactly the same; to add a 4.5xl to the config, which I have added, reluctantly. It seems reasonable that there would be a value between these two points.
